polling_stations/apps/data_finder/tests/test_postcode_builder.py

```diff
@@ -0,0 +1,65 @@
+from django.test import TestCase
+from uk_geo_utils.models import Onspd
+
+from addressbase.models import UprnToCouncil, Address
+from councils.models import Council
+from data_finder.tests.utils import PostcodeBuilder
+
+
+class TestPostcodeTestCase(TestCase):
+    def test_construction(self):
+        postcode = PostcodeBuilder()
+        self.assertIsInstance(postcode.postcode, str)
+        self.assertEqual(len(postcode.councils), 1)
+        self.assertIsInstance(postcode.councils[0], Council)
+        self.assertEqual(postcode.councils[0].council_id[0], "E")
+
+    def test_construction_with_multiple_councils(self):
+        postcode = PostcodeBuilder(multiple_councils=True, nations="W")
```

Comment:
Should this be a list or set? I think it's currently working because a `str` is an iterable, but it looks odd to have a string here and a tuple later

Reply:
I'd settled on passing a tuple as the argument - just missed it in the test. Realise I also miss the trailing comma here: https://github.com/DemocracyClub/UK-Polling-Stations/blob/eea82d38edfcf849e447418b70a300f282681d03/polling_stations/apps/data_finder/tests/utils.py#L44
Perhaps there's a reason not to use a tuple though?
A TODO if we go with this is to add some validation - eg `(W,S)` shouldn't be allowed.

Reply:
Would this also work in the builder pattern, with a `in_wales()` (etc) method?

Reply:
Hmm, yeah that would work, I guess what I'm not sure about is calling a `in_country` method every time, but actually maybe being explicit is quite good. :thinking: